src/OrchardCore.Modules/OrchardCore.Workflows/WorkflowPruning/Services/WorkflowStatusBuilder.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Linq;
+using OrchardCore.Workflows.Models;
+
+namespace OrchardCore.Workflows.WorkflowPruning.Services;
+
+internal sealed class WorkflowStatusBuilder
```

Comment:
this is a bad change, whoever added that.

Orchard is suposed to be an extensible framework, and I have far too much code we have to copy in from Orchard because someone decided to make something internal, and it needed tweaking